src/components/Toast/Toast.tsx

```diff
@@ -51,6 +44,8 @@ const ToastContent = styled(Paragraph)`
   text-decoration: none;
   text-align: left;
 
+  button,
```

Comment:
@dclerici-ssc if you use just `button` as a selector there is no need to also add `button:first/last-child`

Reply:
TBH was to solve the problem with buttons within an ` <Inline>` component because inline apply/force margin to children element. As you can see in the TwoActions Toast Story.

So to solve it, I incremented the specificity with &&& button and it works because it overrides the Inline margin.